README.md

```diff
@@ -149,12 +145,12 @@ The installation prerequisites are the same as described in the TensorFlow
         git checkout v0.6.1
 
 
-7. Next, build and install nGraph bridge. 
+7. Next, build and install nGraph-tf. 
```

Comment:
step 6: `Now clone the `ngraph-tf` repo one level above -- in the *parent* directory of the `tensorflow` repo cloned in step 1`
this requirement is probably no longer there.